recipes/nanopb/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, replace_in_file
+from os import path
+
+class NanoPbConan(ConanFile):
+    name = "nanopb"
+    version = "0.4.8"
+    license = "zlib"
+    url = "https://jpa.kapsi.fi/nanopb/"
```

Comment:
```suggestion
    url = "https://github.com/conan-io/conan-center-index"
    homepage = "https://jpa.kapsi.fi/nanopb/"
```

the `url` attribute is used for the recipe itself, while `homepage` is for the packaged project